data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1625 @@
+				accept
+	on complete
+		conversation
+			`On Viminal you easily locate the maintenance yards and eventually find Taely buried in a Pelican. She spots you approaching and frees herself from the machinery. "Greetings, <first>. How is our station faring?" You reply that it is improving rapidly and hand her the data crystal. "Ah, good. The detailed chemical analysis and profiles." She slots it into her console and taps a few commands. "Okay, I just need to incorporate these profiles into the system and make a few more adjustments. Meet me in the spaceport cafeteria in a couple hours."`
+
+
+
+mission "Remnant: Cognizance 12"
+	name "Meet with escorts"
+	description "Travel with the <npc> to pick up a shipment of atmosphere from <stopovers>, and deliver it to <destination>."
+	cargo equipment 8
+	source "Viminal"
+	destination "Esquiline"
+	stopover "Nasqueron"
+	npc accompany
+		government "Remnant"
+		personality timid launching escort
+		fleet
+			names "remnant"
+			cargo 8 equipment
+			variant
+				Pelican
+				Puffin 4
+	to offer
+		has "Remnant: Cognizance 11: done"
+	on offer
+		conversation
+			`In the cafeteria, you are met by Taely and several other captains. "Thank you for joining us," she signs. "The goal of this mission is to travel to Nasqueron together with equipment designed to harvest some atmosphere from the void sprite's worlds before bringing it back to Esquiline. The equipment on the Puffins will collect, compress, and store the gas, which you can offload to specially prepared tanks on the Pelican. Then proceed to <planet> and offload what you have collected to the station."`
+			branch penguin
+				has "event: remnant: penguin"
+			`	She turns to face the group of captains and gestures with tones of respect. "May the Embers bring interest to your journey."`
+				accept
+			label penguin
+			`	She nods at you and continues. "The Penguin will be even better than the Puffin for this task, and you are welcome to use her, but we do not have any available to accompany you. Continuing to rely on Puffins is not ideal, but we will make do with what we have." She turns to face the group of captains and gestures with tones of respect. "May the Embers bring interest to your journey."`
+				accept
+	on stopover
+		dialog
+			`You cruise through the atmosphere at a relaxed pace, waiting while the equipment does its work. You see fewer void sprites than usual, and many have scars. Once the equipment indicates that it has reached capacity, you climb up to where the Pelican is waiting in the high atmosphere, hook up the connections to offload the gas, and then head back down into the depths of the planet. After several hours, the captain of the Pelican informs you that the tanks are full, and it is time to head to <planet>.`
+	on complete
+		event "remnant: postverta reveal"
+		conversation
```

Comment:
If the block of conversation felt like it was one contiguous set, I kept it as one conversation so that the player didn't have back to back conversation boxes popping up. I personally find that a bit disruptive. It's OK if it is unrelated missions, but annoying when it is two parts of the same mission.